pkg/webhook/validation.go

```diff
@@ -133,8 +133,14 @@ func (v *validator) validateBundle(ctx context.Context, bundle *trustapi.Bundle)
 				unionCount++
 			}
 
+			if source.DefaultPackage {
+				unionCount++
+			}
+
```

Comment:
note to self: courtesy of @inteon : Consider `useDefaultCAs` instead of `defaultPackage` to try to make it clearer at the point of use what's going on.

See also [this comment](https://github.com/cert-manager/trust-manager/pull/77#issuecomment-1355126726)